modules/express/test/index.test.ts

```diff
@@ -20,21 +17,12 @@ import {
   CustomRequestHandler,
   KvsClient,
   Session,
-  EncodedHttpRequest
+  EncodedHttpRequest,
+  GeneralTypeMap
 } from "@phenyl/interfaces";
 
 type Diary = { id: string };
```

Comment:
[imo] since request and response type stays the same, it can be written this way to shorten code at line 104
```suggestion
type Diary = ReqRes<{ id: string }>;
```